docs/pages/07_service.adoc

```diff
@@ -0,0 +1,955 @@
+    [realm "<realm>"]
+    [identity <identity>];
+----
+
+where the <name> is the name of the actor. The <realm> is a security policy domain defined in Keycloak (see Keycloak documentation). Realms  contain a collection of users and can manage and authenticate them. The <identity> is an expression that selects an attribute of the mapped <entity>. The <identity> is used to unambiguously identify the user and define the relationship with the user in the <realm>.
+
+*Example:*
+
+----
+actor SalesPersonActor(SalesPerson salesPerson)
+    realm "COMPANY"
+    identity salesPerson.email ;
+----
+
+The above example defines an actor named `SalesPersonActor`. The company's sales staff are managed in the "COMPANY" Keycloak realm. The salesperson's unique identifier is their email address.
+
+
+=== Granting permissions
+
+The actors within the application can be considered the embodiment of external clients who can use the services granted to them. That is, an actor exposes services to the outside world. 
+
+Defining an actor with no `grants` would let users to authenticate themselves and log into the application. However, they cannot access any resources (services and data) within our application.
+
+The actor object can grant permission to exports containing services. After granting permission to an export, the user can call all the services in the export.
+
+To authorize actors to use services, use `grants` in the actor specification.
+
+*Syntax:*
+
+----
+actor <name> ([<entity> <alias>]) [grants <export>, ...]
+    [realm "<realm>"]
+    [identity <identity>];
+----
+
+where the `grants` keyword is followed by a comma-separated list of exports.
+
+*Example:*
+
+----
+entity Person {
+    field String email ;
+}
+
+transfer PersonTransfer(Person person) {
+    field String email = salesPerson.email ;
+}
+
+export ManagerExport {
+    service static PersonTransfer create() {
+        // implement this body in Java
+    }
+}
+
+actor Manager grants ManagerExport;
+----
+
+The example above allows the manager to create a new person. To grant permissions to external users, we first need to define entities, transport objects and exports. The Manager actor in this application has no realm or identity, meaning this actor is an anonymous actor. Anonymous actors are publicly available, anyone who knows the URL of the Manager actor interface can invoke the granted create service. This way, the Manager actor is given access to the ManagerExport's create service, with which the user can create a PersonTransfer.
+
+*Permission graph*
+
+// https://en.wikipedia.org/wiki/Graph-based_access_control
+
+JUDO's security mechanism is based on a logical permission graph, where the entry point for external users to the application is the actor. Calling the services granted to the actor the user can access transfer objects. Once the user has access to a transfer object, they can call the services granted by the transfer object (and can access further transfer objects).
```

Comment:
Description should contain info about what entities could be paired up in mapped exports and mapped transfer objects. Could we use `grant` with completely different entity-based exports? What about inheritance?